config/dev/openstack-managedcluster.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: hmc.mirantis.com/v1alpha1
+kind: ManagedCluster
+metadata:
+  name: openstack-dev
+  namespace: ${NAMESPACE}
+spec:
+  template: openstack-standalone-cp-0-0-1
+  credential: openstack-cluster-identity-cred
+  config:
+    controlPlaneNumber: ${CONTROL_PLANE_MACHINE_COUNT}
+    workersNumber: ${WORKER_MACHINE_COUNT}
```

Comment:
Let's keep `controlPlaneNumber` and `workersNumber` as 1 for dev purposes to be consistent with all the other dev `ManagedCluster` yamls. But it's not a big deal.

Reply:
I wanted to stay in line with [this](https://cluster-api.sigs.k8s.io/user/quick-start#required-configuration-for-common-providers). But sure, let's keep at 1.